### PR TITLE
DEVOPS - RC1

### PR DESCRIPTION
**Pull request by bsnopek-freshworks**
- Updated deployment configs and params to build all pods together
- Updated deployment-objects for inclusion of services, routes, configmaps, etc
- Updated to connect to EDB - although there is still an issue punching through load-balancer.

### CODE REVIEW

File: devops/openshift/_entity-filer.dc.json [hunk 1]
```diff
@@ -4,18 +4,225 @@
     "metadata": {
         "annotations": {
             "description": "Deployment template for a entity-filer service.",
-            "tags": "${NAME}-${TAG_NAME}"
+            "tags": "${COMP_NAME"
```
Comment (kialj876) [inline devops/openshift/_entity-filer.dc.json]: missed a bracket here

File: devops/openshift/_legal-api.dc.json [hunk 2]
```diff
@@ -109,23 +455,22 @@
                         "type": "ConfigChange"
                     }
                 ],
-                "replicas": "${REPLICAS}",
+                "replicas": "${{REPLICA_MIN}}",
```
Comment (kialj876) [inline devops/openshift/_legal-api.dc.json]: Double brackets!

File: devops/openshift/_legal-api.dc.json [hunk 3]
```diff
@@ -109,23 +455,22 @@
...
-                            "app": "${COMP_NAME}-${SUFFIX}",
-                            "deploymentconfig": "${COMP_NAME}-${SUFFIX}"
+                            "name": "${COMP_NAME}-${SUFFIX}"
                         }
                     },
                     "spec": {
                         "containers": [{
                             "name": "${COMP_NAME}",
-                            "image": "docker-registry.default.svc:5000/${IMAGE_NAMESPACE}/${COMP_NAME}:${SUFFIX}",
+                            "image": " ",
```
Comment (kialj876) [inline devops/openshift/_legal-api.dc.json]: Is this on purpose? Will it create the container without an image somehow?

File: devops/openshift/_entity-filer.dc.json (hunk 1 above)
Comment (bsnopek-freshworks) [reply devops/openshift/_entity-filer.dc.json]: Good catch ! Thanks

File: devops/openshift/_legal-api.dc.json (hunk 2 above)
Comment (bsnopek-freshworks) [reply devops/openshift/_legal-api.dc.json]: Double braces allow for input of non-string values, the quotes will get replaced with the integer value. 

https://docs.openshift.com/container-platform/3.11/dev_guide/templates.html#writing-parameters

File: devops/openshift/_legal-api.dc.json (hunk 3 above)
Comment (bsnopek-freshworks) [reply devops/openshift/_legal-api.dc.json]: This is on purpose yes - the groovy script that actually facilitates the deployment populates that value with properties derived from other variables.

**Comment by rstens** [comment]
No other comments, but I would like a in person walk-through and discussion on this approach.

**Pull request merged**